tests/testthat/test-emmeans.R

```diff
@@ -7,7 +7,7 @@ test_that("emmeans emits a message about mmrm registration on load", {
 
   # Detach emmeans in case it was loaded in session or previous test run
   if (isNamespaceLoaded("emmeans")) {
-    detach("package:emmeans", unload = TRUE)
+    unloadNamespace("emmeans")
```

Comment:
library will attach namespace; even if loaded the namespace can be not attached so can't use detach but better use unload

Reply:
looks good